snap/info.go

```diff
@@ -124,6 +124,7 @@ type SideInfo struct {
 	EditedSummary     string   `yaml:"summary,omitempty" json:"summary,omitempty"`
 	EditedDescription string   `yaml:"description,omitempty" json:"description,omitempty"`
 	Private           bool     `yaml:"private,omitempty" json:"private,omitempty"`
+	SupportURL        string   `yaml:"support-url,omitempty" json:"support-url,omitempty"`
```

Comment:
Indeed, I'm not happy about this either. I can extend prepare-image/seeds (thanks for spotting this). But conceptually it feels like the state is not the best place. I'm struggling to find better alternatives though.